### PR TITLE
Add wordpress importer

### PR DESCRIPTION
**Pull request by akosourov**
Added WP importer for #156.

### CODE REVIEW

File: backend/app/migrator/migrator_test.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ func TestMigrator_ImportDisqus(t *testing.T) {
 		os.Remove("/tmp/disqus-test.xml")
 	}()
 
-	err := ioutil.WriteFile("/tmp/disqus-test.xml", []byte(xmlTest), 0600)
+	err := ioutil.WriteFile("/tmp/disqus-test.xml", []byte(xmlTestDisqus), 0600)
```
Comment (golangci) [inline backend/app/migrator/migrator_test.go]: G303: File creation in shared tmp directory without using ioutil.Tempfile

File: backend/app/migrator/migrator_test.go [hunk 2]
```diff
@@ -39,6 +39,32 @@ func TestMigrator_ImportDisqus(t *testing.T) {
 	assert.Equal(t, 3, len(last), "3 comments imported")
 }
 
+func TestMigrator_ImportWordPress(t *testing.T) {
+	defer func() {
+		os.Remove("/tmp/remark-test.db")
+		os.Remove("/tmp/wordpress-test.xml")
+	}()
+
+	err := ioutil.WriteFile("/tmp/wordpress-test.xml", []byte(xmlTestWP), 0600)
```
Comment (golangci) [inline backend/app/migrator/migrator_test.go]: G303: File creation in shared tmp directory without using ioutil.Tempfile

File: backend/app/migrator/disqus.go [hunk 3]
```diff
@@ -73,7 +73,7 @@ func (d *Disqus) Import(r io.Reader, siteID string) (size int, err error) {
 
 	log.Printf("[DEBUG] imported %d comments to site %s", passed, siteID)
 
-	if failed > 0 && passed == 0 {
+	if passed == 0 {
```
Comment (umputun) [inline backend/app/migrator/disqus.go]: any particular reason why 0 imported should be an error? In addition, this code was forgiving a little bit by design to report as failed, only if nothing were imported.

File: backend/app/migrator/wordpress.go [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+	}
+
+	if failed > 0 {
+		return passed, errors.Errorf("failed to save %d comments", failed)
+	}
+
+	log.Printf("[DEBUG] imported %d comments to site %s", passed, siteID)
+
+	if passed == 0 {
+		err = errors.New("import failed")
```
Comment (umputun) [inline backend/app/migrator/wordpress.go]: same as above. I don't see why nothing imported should be an error.

**Comment by umputun** [comment]
Looks like some of the tests hanging. Not sure if related to your changes or something on travis side

**Comment by umputun** [comment]
One more thing - you missed readme update. We need a new section similar to "Initial import from Disqus"

**Comment by umputun** [comment]
well, looks like something in your PR causing this hanging test. I have tried to restart your build twice and it is hanging. At the same time, master build passed just fine.

Tried to run your tests locally and it worked fine, but something here driving travis crazy.

**Comment by umputun** [comment]
another possibility - some test running before yours kept things in bad shape. I have found `migrator.remark_test.go" doesn't remove bolt files, fixed in master. You may try to merge to yours and see if helped

File: backend/app/migrator/disqus.go (hunk 3 above)
Comment (akosourov) [reply backend/app/migrator/disqus.go]: It looks like `if failed > 0 && passed == 0` never be executed until `if failed > 0 return ...` exists upper. Maybe swap this conditions? I mean first put `if failed > 0 && passed == 0` and then put `if failed > 0 return ...`
So when `failed` exists it should return error but differentiate cases when nothing were converted and parts were converted.

My verstion was if nothing were converted it should return error regardless of `failed` exists or nor. Yeah, I agree it is incorrect

**Comment by akosourov** [comment]
> One more thing - you missed readme update. We need a new section similar to "Initial import from Disqus"

Of course, I'll add it at this evenening/night

> You may try to merge to yours and see if helped

Ok, I'll try. thanks

**Comment by akosourov** [comment]
@umputun
1. I've added WP section in readme file. But I haven't got a WP site, so this instruction can be not so strict. So I'll ask the author of ticket to correct instruction and leave feed back about WP importer.
2. I've changed error return at disqus.go and wordpress.go.
Please review

**Pull request merged**